docs/qiita-16S-analysis.rst

```diff
@@ -3,218 +3,424 @@
+
+To perform a Bray-Curtis beta diversity analysis, select the rarefied "dflt_name - BIOM" artifact in the processing network and select "Process". Then select the "dflt_name - BIOM" artifact and select "beta_diversity" from the drop-down menu. The parameters will appear below the workflow diagram:
+
+.. figure::  images/bray_curtis_beta_diversity.png
+   :align:   center
+
+Several parameters have been automatically selected for you. In the "p-metric" field we will specify the beta diversity analysis to run. Enter "braycurtis" in this box, and click "Add Command".
+
+To create a Bray-Curtis dissimilarity principal coordinate plot, select select "pcoa" from the drop-down menu. The parameters will appear below the workflow diagram:
+
+.. figure::  images/bray_curtis_pcoa.png
+   :align:   center
+
+All of the parameter have automatically selected for you just click "Add Command". Once the command is added the workflow should appear as follows:
+
+.. figure::  images/bray_curtis_workflow.png
+   :align:   center
+
+Click the run button to start the process of the beta diversity analysis. The view will return to the original screen, while the beta diversity analysis job runs.
+
+Another tool for measuring beta diversity is unweighted beta diversity analysis. Unweighted beta diversity analysis is when the types but not quantity of each taxa is taken into consideration when comparing samples to one another. This differs from weighted analysis which takes into consideration both the amount and variety of taxa in a sample. 
+
+To perform an unweighted beta diversity analysis, select the rarefied "dflt_name - BIOM" artifact in the processing network and select "Process". Then select the "dflt_name - BIOM" artifact and select "beta_diversity" from the drop-down menu. The parameters will appear below the workflow diagram:
+
+.. figure::  images/unweighted_beta_diversity.png
+   :align:   center
+   
+Several parameters have been automatically selected for you. In the "p-metric" field enter "unweighted Unifrac" and in the "i-tree" field enter "/databases/gg/13_8/trees/97_otus.tree", and click "Add Command". 
+
+To create an unweighted beta analysis principal coordinate plot, select select "pcoa" from the drop-down menu. The parameters will appear below the workflow diagram:
+
+.. figure::  images/unweighted_pcoa.png
+   :align:   center
+
+All of the parameters have been automatically selected for you just click "Add Command". Once the command is added the workflow should appear as follows:
+
+.. figure::  images/unweighted_workflow.png
+   :align:   center
+
+Click the run button to start the process of the beta diversity analysis. The view will return to the original screen, while the beta diversity analysis job runs.
+
+A final way to analyze the beta diversity is through filtered unweighted beta diversity analysis. This allows you to filter out samples. 
```

Comment:
Ya, this is "how to run Qiita" with a bit of context. The workshop is more for what it means. If we want a guide to interpreting data later, which would be cool, then we can make it and reference back to the processing and analysis guides if people need guidance on getting the figures to interpret.